src/test/scala/com/mozilla/telemetry/TestUtils.scala

```diff
@@ -86,14 +86,15 @@ object TestUtils {
     }
   }
 
-  def generateMainMessages(size: Int, fieldsOverride: Option[Map[String, Any]]=None, timestamp: Option[Long]=None): Seq[Message] = {
+  def generateMainMessages(size: Int, fieldsOverride: Option[Map[String, Any]]=None, timestamp: Option[Long]=None,
+                           removeDisplayVersion: Boolean=false): Seq[Message] = {
```

Comment:
Can we change this to a `removeFields: List[String]`, which are fields to remove? The match below would just look like `case JField(x, _) if removeFields.contains(x) => true`.

Reply:
Sure, I did this way because there is only one field we are currently removing.

Reply:
I wound up naming the variable 'fieldsToRemove' instead